Source/CNTK/CNTK.cpp

```diff
@@ -690,7 +691,7 @@ int wmainOldCNTKConfig(int argc, wchar_t* argv[])
 
 #ifndef CPUONLY
     ConfigValue val = config("deviceId", "auto");
-    if (!EqualCI(val, "cpu") && !EqualCI(val, "auto"))
+    if (!EqualCI(val, "cpu") && !EqualCI(val, "auto") && !EqualCI(val, "localRank"))
```

Comment:
User will have two choice here.
1. use 'auto' mode in at least one of the 2 jobs.
2. if they know their machine well, they can run a job on any combinations of gpu they want manually with CUDA_VISIBLE_DEVICES and localRank mode